src/lib.rs

```diff
@@ -166,7 +166,7 @@ where
     /// Verifies device to address 0x68 with WHOAMI.addr() Register
     fn verify(&mut self) -> Result<(), Mpu6050Error<E>> {
         let address = self.read_byte(WHOAMI)?;
-        if address != WHO_AM_I_VALUE {
+        if WHOAMI_VALUES.contains(&address) {
```

Comment:
I think this is somewhat dangerous.
The old code only allows specified chip (e.g. when Mpu6886 type is used, only MPU6886 chip can be used). However, because the new code does not use WHO_AM_I_VALUE specified as a type parameter, it does not produce error when the connected chip is different from expected one.

Reply:
Because MPU9250, 6886, and 6050 have slightly different functions, raising error makes debugging easier (in my opinion).